ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Client.scala

```diff
@@ -146,7 +146,7 @@ private[ember] class H2Client[F[_]: Async](
             case Some("http/1.1") => Resource.pure[F, SocketType](Http1)
             case Some(other) =>
               Resource.raiseError[F, SocketType, Throwable](
-                new Throwable("Unknown Protocol Received")
+                new ProtocolException(s"Unknown protocol")
```

Comment:
```suggestion
                new ProtocolException("Unknown protocol")
```

Don't need the string interpolator without that other.

Reply:
Oops.  Caught me adding it and then having second thoughts.